.expeditor/scripts/release_habitat/update_api_docs_dev.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
I don't see anything referencing this script. We should remove it.

Reply:
It is a standalone script to run in the dev environment only.  I did not want to make the update_api_docs.sh convoluted with dev logic but still wanted to be able to test things out on my local dev setup.